PostReleaseActivities.md

```diff
@@ -0,0 +1,26 @@
+﻿Post-release activities
+=================================================================
+
+Please follow the below steps after publishing analyzer NuGet packages from this repo onto NuGet.org:
+
+1. Create a new release:
+   1. Draft: Click [here](https://github.com/dotnet/roslyn-analyzers/releases/new) to draft a new release. For reference, you can look at any of the existing releases, say [v2.9.3](https://github.com/dotnet/roslyn-analyzers/releases/edit/v2.9.3).
+   2. Release notes: Follow the steps in the *Steps to generate Release Notes* below to generate Release notes and copy the generated notes to the description section of the new release.
+   3. Publish: Mark the release as a pre-release if appropriate and click "Publish Release".
+2. Repo changes:
+   1. Checkout a new branch from latest sources of release branch.
+   2. Update `VERSIONING.md`: Add a new row in the released version table.
+   3. Update `.github\ISSUE_TEMPLATE.md`: Update the package version in the example section to the latest released package version.
+   4. Update `eng\Versions.props`:
+      1. Bump up the `VersionPrefix`.
+      2. Reset `PreReleaseVersionLabel` to `beta1`.
+      3. Update `MicrosoftCodeAnalysisFXCopAnalyersVersion` to the latest released package version. Build the repo by invoking `Build.cmd` and fix/suppress any new CA diagnostics, as appropriate.
+   5. Create and submit a PR with the above changes.
+
+Steps to generate Release Notes
```

Comment:
@dotpaul Below steps helped me generate release notes for [v2.9.4 release](https://github.com/dotnet/roslyn-analyzers/releases/tag/v2.9.4). Do the steps seem correct/minimal to you? Might also be good to include restoring/building the project as part of `GenDiffNotes.cmd`? Not sure if it can be simplified further.

Reply:
Hrm, I guess I should've added it to the solution so people don't have to build to tool by itself.

Did you have to run the cmd script from an admin command prompt?  I don't think I did.

Also, for keeping track of fixes / changes that are manually noted, we could use a draft release.

Reply:
> Did you have to run the cmd script from an admin command prompt? I don't think I did.

I always work off an admin developer command prompt, so I will take your word. Feel free to update this section with a follow-up PR.